lib/_stream_wrap.js

```diff
@@ -16,26 +16,26 @@ function StreamWrap(stream) {
   this._list = null;
 
   const self = this;
-  handle.close = function(cb) {
+  handle.close = (cb) => {
     debug('close');
     self.doClose(cb);
```

Comment:
since these are being changed to arrow functions, the `self` can be switched to `this`